pkg/plugins/manager/loader/assetpath/assetpath.go

```diff
@@ -153,3 +153,18 @@ func getBaseDir(pluginDir string) string {
 	}
 	return baseDir
 }
+
+func (s *Service) GetLocales(n PluginInfo) (map[string]string, error) {
+	pathToLocales, err := s.RelativeURL(n, "locales/")
```

Comment:
this will use a CDN path if CDN is supported by the plugin. Is it expected or should it skip the CDN evaluation before it's supported?

Reply:
It it expected to find the translation files on the CDN if the CDN is supported.